packages/rulesets/src/spectral/functions/response-schema-specified-for-success-status-code.ts

```diff
@@ -0,0 +1,21 @@
+export const responseSchemaSpecifiedForSuccessStatusCode = (putOperation: any, _opts: any, ctx: any) => {
+  const errors = []
+  const path = ctx.path
+  const successCodes = ["200", "201"]
+
+  for (const code of successCodes) {
+    if (putOperation.responses[code]) {
+      const response = putOperation.responses[code]
+      if (!response?.schema) {
```

Comment:
should report error only when 201 is present and schema is empty,

Reply:
should we not check for 200?

Reply:
Oh, I've updated the comment.

Reply:
I am checking if the code 200/201 is present in line 7
and if code is present and schema isnt present (checked in line 9), we throw error right?

Reply:
Okay, the 'response?.schema' misled me.
```suggestion
      if (!response.schema) {
```